src/pages/dashboard/Dashboard.tsx

```diff
@@ -106,22 +106,13 @@ const Dashboard = () => {
       [products, party]
     ) ?? [];
 
-  const delegableProducts: Array<Product> = activeProducts.filter((p) =>
+  const authorizedDelegableProducts = activeProducts.filter((activeProduct) =>
```

Comment:
- Leaving the typing is a great habit especially when using TypeScript, restore it
- When we use javascript methods such as filter, map etc.. we use, where possible, a few letters (e.g. activeProduct -> ap)